---
# Documentation: https://wowchemy.com/docs/managing-content/

title: 'Comparative analyses of Mikania (Asteraceae: Eupatorieae) plastomes and impact
  of data partitioning and inference methods on phylogenetic relationships'
subtitle: ''
summary: ''
authors:
- Verônica A Thode
- Caetano T Oliveira
- Benoı̂t Loeuille
- Carolina M Siniscalchi
- José R Pirani
tags: []
categories: []
date: '2021-06-01'
lastmod: 2021-07-17T15:20:02-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: [compositae]
publishDate: '2021-07-17T20:20:02.233981Z'
publication_types:
- '2'
abstract: 'We assembled new plastomes of 19 species of Mikania and of Ageratina fastigiata,
  Litothamnus nitidus, and Stevia collina, all belonging to tribe Eupatorieae (Asteraceae).
  We analyzed the structure and content of the assembled plastomes and used the newly
  generated sequences to infer phylogenetic relationships and study the effects of
  different data partitions and inference methods on the topologies. Most phylogenetic
  studies with plastomes ignore that processes like recombination and biparental inheritance
  can occur in this organelle, using the whole genome as a single locus. Our study
  sought to compare this approach with multispecies coalescent methods that assume
  that different parts of the genome evolve at different rates. We found that the
  overall gene content, structure, and orientation are very conserved in all plastomes
  of the studied species. As observed in other Asteraceae, the 22 plastomes assembled
  here contain two nested inversions in the LSC region. The plastomes show similar
  length and the same gene content. The two most variable regions within Mikania are
  rpl32-ndhF and rpl16-rps3, while the three genes with the highest percentage of
  variable sites are ycf1, rpoA, and psbT. We generated six phylogenetic trees using
  concatenated maximum likelihood and multispecies coalescent methods and three data
  partitions: coding and non-coding sequences and both combined. All trees strongly
  support that the sampled Mikania species form a monophyletic group, which is further
  subdivided into three clades. The internal relationships within each clade are sensitive
  to the data partitioning and inference methods employed. The trees resulting from
  concatenated analysis are more similar among each other than to the correspondent
  tree generated with the same data partition but a different method. The multispecies
  coalescent analysis indicate a high level of incongruence between species and gene
  trees. The lack of resolution and congruence among trees can be explained by the
  sparse sampling (~ 0.45% of the currently accepted species) and by the low number
  of informative characters present in the sequences. Our study sheds light into the
  impact of data partitioning and methods over phylogenetic resolution and brings
  relevant information for the study of Mikania diversity and evolution, as well as
  for the Asteraceae family as a whole.'
publication: '*Sci. Rep.*'
doi: https://doi.org/10.1038/s41598-021-92727-6
---
