---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Using genomic data to develop SSR markers for species of Chresta (Vernonieae,
  Asteraceae) from the Caatinga
subtitle: ''
summary: ''
authors:
- Carolina M Siniscalchi
- Benoit Loeuille
- Jos√© R Pirani
- Jennifer R Mandel
tags: []
categories: []
date: '2019-12-01'
lastmod: 2021-07-17T15:20:05-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: [compositae]
publishDate: '2021-07-17T20:20:05.565247Z'
publication_types:
- '2'
abstract: Chresta is a genus mostly endemic to Brazil that presents several rupicolous
  species with naturally fragmented distributions. Aiming to facilitate studies about
  genetic diversity and structure in these species, we developed a set of 22 nuclear
  and 6 plastid microsatellite markers that are transferable among different species
  of the genus. We used previously obtained genomic data from target capture and Illumina
  sequencing to identify putative repeat regions, designed and synthesized primers,
  and genotyped individuals from different populations of three species. All loci
  were successfully amplified in all three species and were overall variable, except
  for the plastid markers, which were monomorphic in two species. These newly developed
  microsatellites will be useful in studies focusing on the population genetics of
  Chresta.
publication: '*Rev. Bras. Bot.*'
doi: https://doi.org/10.1007/s40415-019-00569-4
---
